src/components/CustomerQuota/ItemsSelection/ItemCheckbox.tsx

```diff
@@ -32,7 +34,15 @@ export const ItemCheckbox: FunctionComponent<{
           showAddons={isShowAddonsItems}
         />
       }
-      addons={<AddonsItems />}
+      addons={
+        isChargeable ? (
+          <AddonsItems
+            ids={ids}
+            isShowAddonItems={isShowAddonsItems}
+            categoryFilter={[category]}
```

Comment:
Given that this will always be 1 item, do we need a list?

Reply:
this one is from @yongjiajie 's pr. I think he meant to make it generic so that we can filter multiple categories if needed. What do you think?

Reply:
Yea the API remains, just that on this particular component it should only have 1 category

Reply:
oh ok, understood your comment